rgw/admin/admin_bucket.go

```diff
@@ -0,0 +1,32 @@
+//go:build ceph_preview
+// +build ceph_preview
+
+package admin
```

Comment:
Since all new apis need to be added as separate file, I recommend filename should be similar to API like `list_bucket_with.go` than admin_bucket.go

Reply:
Done.